notifyrules/SaveDatabaseAction.php

```diff
@@ -3,6 +3,7 @@
 use Ramsey\Uuid\Uuid;
 use RainLab\Notify\Classes\ActionBase;
 use Illuminate\Database\Eloquent\Model as EloquentModel;
+use October\Rain\Exception\ApplicationException;
```

Comment:
```suggestion
use ApplicationException;
```

Reply:
Good to know that ApplicationException is an alias, I always used the full namespace because of the autocomplete. Sorry.